README.md

```diff
@@ -1,2 +1,70 @@
-# React-Calculator
-This is a single page app (SPA) calculator made with React so everyone can do their calculations.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
```

Comment:
- [ ] Please add details to your readme file like (description, project setup, Author name, etc....). and remove the old readme file please.

Reply:
Done! Thanks for pointing out! 👍